selfdrive/ui/qt/setup/setup.cc

```diff
@@ -363,6 +364,25 @@ void Setup::nextPage() {
 }
 
 Setup::Setup(QWidget *parent) : QStackedWidget(parent) {
+}
+
+void Setup::showEvent(QShowEvent *event) {
+  static bool initialized = false;
+  if (std::exchange(initialized, true) == true) return;
+
+  // select language
+  QString currentLang = "main_en";
```

Comment:
I think I like no language selected better. If you want English, you have to press Cancel to proceed like this

Reply:
yes i didn't notice the problem